continuous_integration/Dockerfile

```diff
@@ -28,16 +30,22 @@ COPY --chown=coder:coder . /home/coder/legate
 
 RUN chmod a+x /home/coder/.local/bin/* && \
     mkdir -p /tmp/out && \
-    chown -R coder:coder /tmp/out  && \
-    chown -R coder:coder /home/coder/.artifacts
+    chown -R coder:coder /tmp/out && \
+    sudo chown -R coder:coder /home/coder
```

Comment:
Why is that necessary? Are the permissions wrong to begin with?

Reply:
The ~/.artifacts folder gets mounted with root as owner. This simply changes the ownership back to coder.

Reply:
Line removed.